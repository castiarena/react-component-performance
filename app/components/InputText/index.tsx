import React from 'react';
import styled from 'styled-components';

const InputText = styled.input``;

export default InputText;
