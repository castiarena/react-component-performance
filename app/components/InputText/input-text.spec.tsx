import React from 'react';
import 'jest-styled-components';
import InputText from '.';
import renderWithTheme from '../../testing-tools/renderWithTheme';

describe('InputText test suite', () => {
    it('should a default InputText', () => {
        const { asFragment } = renderWithTheme(
            <InputText defaultValue="something" type="text" />,
        );
        const fragment = asFragment();

        expect(fragment).toMatchSnapshot();
    });
});
