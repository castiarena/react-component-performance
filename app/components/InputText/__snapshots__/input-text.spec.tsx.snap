// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputText test suite should a default InputText 1`] = `
<DocumentFragment>
  <input
    class=""
    type="text"
    value="something"
  />
</DocumentFragment>
`;
